### PR TITLE
Adding extra files

### PR DESCRIPTION
**Pull request by mbalyi**
Comment about what you did.

Reasons of changes.

Additional comments:
- Review comments
- Test results
- Reports
- Links
- Linked issue: #1

### CODE REVIEW

File: prime-numbers.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+     while True:
+         # get the first number from the iterator (always a prime)
+         prime = next(numbers)
+         yield prime
+
+         # this code iteratively builds up a chain of
+         # filters...slightly tricky, but ponder it a bit
+         numbers = filter(prime.__rmod__, numbers)
+
+for p in iter_primes():
```
Comment (mbalyi) [inline prime-numbers.py:17]: itt is gond van

**Comment by zubi76** [comment]
megoldódott

File: prime-numbers.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import itertools
+
+def iter_primes():
+     # an iterator of all numbers between 2 and +infinity
```
Comment (mbalyi) [inline prime-numbers.py:4]: Adding new comment

File: prime-numbers.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+     # generate primes forever
+     while True:
+         # get the first number from the iterator (always a prime)
+         prime = next(numbers)
+         yield prime
+
+         # this code iteratively builds up a chain of
+         # filters...slightly tricky, but ponder it a bit
+         numbers = filter(prime.__rmod__, numbers)
```
Comment (mbalyi) [inline prime-numbers.py:15]: new comment

File: prime-numbers.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import itertools
+
+def iter_primes():
```
Comment (nlevhu) [inline prime-numbers.py:3]: Review test

File: prime-numbers.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+     # an iterator of all numbers between 2 and +infinity
+     numbers = itertools.count(2)
+
+     # generate primes forever
+     while True:
```
[the remark this answers is not in the document]
Comment (LaszloBakiFlow) [reply prime-numbers.py:8]: comment

File: prime-numbers.py [hunk 6]
```diff
@@ -0,0 +1,20 @@
+import itertools
+
+def iter_primes():
+     # an iterator of all numbers between 2 and +infinity
+     numbers = itertools.count(2)
```
Comment (zubi76) [inline prime-numbers.py:5]: new review
let's finish here

File: prime-numbers.py (hunk 4 above)
Comment (nlevhu) [inline prime-numbers.py:3]: Test2

File: prime-numbers.py (hunk 3 above)
Comment (nlevhu) [inline prime-numbers.py:15]: Test 3

File: prime-numbers.py [hunk 7]
```diff
@@ -0,0 +1,20 @@
+import itertools
+
```
Comment (gabi9669) [inline prime-numbers.py:2]: comment

**Review by nlevhu — APPROVED** [review]
Approved!

File: prime-numbers.py (hunk 1 above)
Comment (LaszloBakiFlow) [reply prime-numbers.py:17]: Csak volt.

**Review by LaszloBakiFlow — APPROVED** [review]

**Review by zubi76 — APPROVED** [review]

**Comment by padrienn03** [comment]
review test

File: prime-numbers.py [hunk 8]
```diff
@@ -0,0 +1,20 @@
+import itertools
```
Comment (gabi9669) [inline prime-numbers.py:1]: Comment

File: queens-problem.py [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+                return test_queens
+            else:
+                return add_queen(test_queens)
+        except BailOut:
+            pass
+    raise BailOut
+
+queens = add_queen([])
+print (queens)
+print ("\n".join(". "*q + "Q " + ". "*(BOARD_SIZE-q-1) for q in queens))
```
Comment (gabi9669) [inline queens-problem.py:28]: comment?

**Review by gabi9669 — APPROVED** [review]

File: prime-numbers.py (hunk 5 above)
[the remark this answers is not in the document]
Comment (AxlAlmasi) [reply prime-numbers.py:8]: Ide kell írni?

[the remark this answers is not in the document]
Comment (mbalyi) [reply prime-numbers.py:8]: yes

**Pull request merged**